trollpy/tests.py

```diff
@@ -0,0 +1,357 @@
+"""Tests for TrollyPy App."""
+import pytest
+import transaction
+import os
+from pyramid import testing
+from .models.meta import Base
+from .models import (
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+    User,
+    KillScore,
+    Audio
+)
+from gtts import gTTS
+
+
+@pytest.fixture(scope="function")
+def sqlengine(request):
+    """Return sql engine."""
+    config = testing.setUp(settings={
+        'sqlalchemy.url': os.environ["DATABASE_URL"]
+    })
+    config.include(".models")
+    settings = config.get_settings()
+    engine = get_engine(settings)
+    Base.metadata.create_all(engine)
+
+    def teardown():
+        testing.tearDown()
+        transaction.abort()
+        Base.metadata.drop_all(engine)
+
+    request.addfinalizer(teardown)
+    return engine
+
+
+@pytest.fixture(scope="function")
+def new_session(sqlengine, request):
+    """Return new session."""
+    session_factory = get_session_factory(sqlengine)
+    session = get_tm_session(session_factory, transaction.manager)
+
+    def teardown():
+        transaction.abort()
+
+    request.addfinalizer(teardown)
+    return session
+
+
+@pytest.fixture
+def dummy_request(new_session, method="GET"):
+    """Instantiate a fake HTTP Request, complete with a database session."""
+    request = testing.DummyRequest()
+    request.method = method
+    request.dbsession = new_session
+    return request
+
+
+@pytest.fixture
+def dummy_post_request(new_session, method="POST"):
+    """Instantiate a fake HTTP Request, complete with a database session."""
+    request = testing.DummyRequest()
+    request.method = method
+    request.dbsession = new_session
+    return request
+
+
+@pytest.fixture
+def testapp():
+    """Create an instance of webtests TestApp for testing routes."""
+    from webtest import TestApp
+    from trollpy import main
+
+    app = main({}, **{'sqlalchemy.url': os.environ["DATABASE_URL"]})
```

Comment:
This won't matter because inside of your `main` function you overwrite the settings dict anyway